db/blip_sync_context.go

```diff
@@ -585,6 +601,13 @@ func (bsc *BlipSyncContext) sendNoRev(sender *blip.Sender, docID, revID string,
 	if !bsc.sendBLIPMessage(sender, noRevRq.Message) {
 		return ErrClosedBLIPSender
 	}
+	messageCPUtime := time.Since(startTime).Milliseconds()
+	messBytes, err := noRevRq.Body()
+	if err == nil && bsc.blipContextDb != nil {
+		bytes := len(messBytes)
+		stat := CalculateComputeStat(int64(bytes), messageCPUtime)
+		bsc.blipContextDb.DbStats.Database().SyncProcessCompute.Add(stat)
+	}
```

Comment:
This code is called from `sendBatchOfChanges` - I wonder if this whole code is going to have such a short period of execution time whether it's worth tracking this separately from sendBatchOfChanges. I'd be interested in someone else's call on this.

Reply:
I think this is called from `sendRevision` which in turn is called from `handleChangesResponse` so I feel like is needed even though I agree it will be very small.

Reply:
SGTM, I think just write a comment about it.